src/Evolution/Executables/GeneralizedHarmonic/EvolveGeneralizedHarmonicNoHorizon.hpp

```diff
@@ -0,0 +1,461 @@
+// Distributed under the MIT License.
```

Comment:
I suggest to instead rename the current EvolveGeneralizedHarmonic to EvolveGeneralizedHarmonicWithHorizon, and then call the new (no-horizon) executable simply EvolveGeneralizedHarmonic. I think this makes it easier to understand what's going on.